src/shared/constants/common.constants.ts

```diff
@@ -17,6 +17,23 @@ export const supportedLocaleList: ESupportedLocales[] = [
   ESupportedLocales.INTERSLAVIC
 ];
 
+export enum EAssetType {
```

Comment:
Enums shouldn't be really be used as a way to record "const" data such as labels as here. They ideally just classify something of what type it is from some constrained set of options.

In that case, we would move this to a .type file such as metadata.types.ts

Another reason for the EAssetType and AssetTypeLabel being needed as a separate const is that this is a string which also need to be in the i18n form.